ggshield/core/lines.py

```diff
@@ -7,8 +7,8 @@
 from ggshield.utils.git_shell import Filemode
 
 
-REGEX_PATCH_HEADER = re.compile(
-    r"^(?P<line_content>@@ -(?P<pre_index>\d+),?\d* \+(?P<post_index>\d+),?\d* @@(?: .+)?)"  # noqa
+REGEX_HUNK_HEADER = re.compile(
+    r"^@@ -(?P<pre_index>\d+),?\d* \+(?P<post_index>\d+),?\d* @@(?P<trailing_content> .+)?"  # noqa
```

Comment:
```suggestion
    r"^@@ -(?P<pre_index>\d+),?\d* \+(?P<post_index>\d+),?\d* @@(?P<trailing_content>.+)?"  # noqa
```
Is there constantly a space here? Otherwise, this could also create an error I guess

Reply:
Yes, the space is always there.